test/integration/Utils.kt

```diff
@@ -29,6 +29,7 @@ import com.vaticle.typedb.client.TypeDB
 import com.vaticle.typedb.client.api.TypeDBOptions
```

Comment:
This is a really messy diff to try and read, but we've just taken our Utils standalone functions and variables, put them in an object and bumped the delays.